aiida/repository/backend/disk_object_store.py

```diff
@@ -156,24 +157,29 @@ def maintain( # type: ignore[override] # pylint: disable=arguments-differ,too-ma
     ) -> dict:
         """Performs maintenance operations.
 
-        :param live:if True, will only perform operations that are safe to do while the repository is in use.
-        :param pack_loose:flag for forcing the packing of loose files.
-        :param do_repack:flag for forcing the re-packing of already packed files.
-        :param clean_storage:flag for forcing the cleaning of soft-deleted files from the repository.
-        :param do_vacuum:flag for forcing the vacuuming of the internal database when cleaning the repository.
-        :param compress:flag for compressing the data when packing loose files.
-        :return:a dictionary with information on the operations performed.
+        :param live: if True, will only perform operations that are safe to do while the repository is in use.
+        :param pack_loose: flag for forcing the packing of loose files.
+        :param do_repack: flag for forcing the re-packing of already packed files.
+        :param clean_storage: flag for forcing the cleaning of soft-deleted files from the repository.
+        :param do_vacuum: flag for forcing the vacuuming of the internal database when cleaning the repository.
+        :param compress: flag for compressing the data when packing loose files. Set to ``Compress.AUTO`` if ``True``.
```

Comment:
Ok, I guess we only allow fully uncompressed or auto compression (no options for YES or KEEP), right?
I guess it's OK for the AiiDA interface in order to keep it simple and we can always point people to the Disk-ObjectStore API for more (or expand this API in the future)

Reply:
Yes, we currently just have a boolean flag because that was all that was supported by the disk objectstore anyway. Don't think we should start making it even more complex now.